common/reflect_resource_test.go

```diff
@@ -301,15 +364,15 @@ func TestPrimitiveReflectValueFromInterface(t *testing.T) {
 
 func TestIterFields(t *testing.T) {
 	v := reflect.ValueOf("x")
-	err := iterFields(v, []string{"x"}, scm, nil)
+	err := iterFields(v, []string{"x"}, scm, map[string]string{}, nil)
```

Comment:
Since we're only reading from this map, we can pass `nil` instead of `map[string]string{}` everywhere.